LectureNotes/Lectures/Fall2018/Lecture6/ASPlecturenotes.tex

```diff
@@ -0,0 +1,142 @@
+\documentclass[12pt]{article}
+
+\usepackage[utf8]{inputenc} %For non-ASCII characters%
+\usepackage[margin=1in]{geometry} %For margins and related things%
+\usepackage{amssymb, amsthm, amsmath, bbm}
+
+\let\oldphi\phi
+\renewcommand{\phi}{\varphi}
+
+\newenvironment{soln}{\begin{proof}[\textsc{Solution}]}{\renewcommand{\qedsymbol}{$\blacklozenge$}\end{proof}}
+
+\title{Poisson Point Processes Cont.}
+\author{Erin McCarthy\thanks{\textbf{email:} emccarth@uoregon.edu}}
+\date{\today}
+\begin{document}
+\maketitle
+
+\section{The Cauchy Distribution}
+
+$C \sim Cauchy(1)$ 
+if $\mathbb{P}\{c \in dx\}= \dfrac{dx}{\pi(1+x)^2}$ and $t \times C \sim Cauchy(t)$
+
+\noindent\rule{4cm}{0.4pt} 
+
+\noindent$N \sim PPP(\dfrac{dtdx}{|x|^2})$ on $[0,\infty) \times \{ \mathbb{R}\setminus \{0\}\}$
+
+\noindent$Let C_t = ($sum of x-values of points of $[0,t] \times \mathbb{R})= \sum x \mathbbm{1}_{[0,t)}(t) dN(t,x)$ 
+
+\subsection*{Claim}
+$\dfrac{1}{t} C_t \stackrel{d}{=} C_1$
+
+\begin{proof}
+If $N = \sum \delta_{(t_i,x_i)} \Rightarrow \dfrac{1}{t} C_t = \sum_{i:t_i\leq1} \dfrac{x_i}{t}$ but \~N $= \sum_i \delta_{\left(\frac{t_i}{t_1}, \frac{x_i}{t}\right)} \sim PPP(\dfrac{dtdx}{x^2})$ i.e. $N \stackrel{d}{=} $\~N
+(since $\dfrac{dsdx}{x^2} = \dfrac{tdu tdv}{t^2dv}=\dfrac{dudv}{v^2}$, where $u=\frac{s}{t}$ and $v=\frac{x}{t}$), so $\dfrac{1}{t}C_t = \int x \mathbbm{1}_{[0,1]}(t) d\text{\~{N}} (t,x) = \text{\~C}(1)$ 
+\end{proof}
+
+
+\subsection*{Consequence}
+Let $x_1,x_2,...,x_n$ iid $Cauchy(1)$ then $\bar{x} = \dfrac{1}{n} (x_1+...+x_n) \sim Cauchy(1)$. This implies that that taking more samples does not tell you more about the mean! But ...\\
+$$\begin{aligned}
+var[\bar{x}] = \frac{1}{n}var[x_1] = \int_0^1 \int_{-\infty}^{\infty} x^2 \frac{dxdt}{x^2} = \infty \\ \text{ even } \mathbb{E}[x_1]=\int_0^1\int_{-\infty}^\infty x \frac{dxdt}{x^2} = \text{ UND }\\
+\end{aligned}$$
```

Comment:
Any words about what this means?

Reply:
I am not sure. Since the points are inf far from the mean is und so the more samples we take we will end up with the same conclusion. But I am not sure, I vaguely remember a statement in class at this point but I cant remember and I did not write it down.